### DISCUSSION TITLE
[ 功能建议 ] SQL上线工单排队进行执行

### DISCUSSION THREAD

**Discussion by shengminjie** [hhyo/Archery#1581] [General]
### 目前遇到的问题/使用障碍
线上环境同时处理多个工单可能会有影响，特别是大表变更调用pt-osc等工具的情况下，建议增加一个排队执行的功能。

### 希望如何解决/实现它
程序逻辑，只同时执行x个任务，如1个任务，当前有任务未完成时，新的任务等待上一个任务完成后才继续执行。

### 其他信息
如果有其他类似的产品功能或者图片信息，可在此提交

**Comment by hhyo** [comment]
这种需要限制同实例串行，非同实例可以并行
